src/renderer/account/components/AccountPanel/TokenBalance/index.js

```diff
@@ -1 +1,22 @@
-export { default } from './TokenBalance';
+import { compose } from 'recompose';
+import { withData, withActions } from 'spunky';
+
+import authActions from 'login/actions/authActions';
+import claimActions from 'shared/actions/claimableActions';
+import withLoadingProp from 'shared/hocs/withLoadingProp';
+import withNetworkData from 'shared/hocs/withNetworkData';
+
+import TokenBalance from './TokenBalance';
+
+const mapAuthDataToProps = ({ address }) => ({ address });
+
+const mapClaimActionsToProps = (actions, props) => ({
```

Comment:
```suggestion
const mapClaimActionsToProps = (actions, { net, address }) => ({
  getBalances: () => actions.call({ net, address })
});
```
Little nitpick

Reply:
not sure how this suggestion thing works, it seems to insert multiple lines 😂

Reply:
Suggestion change beta, nice!  I thought about this, but when I've done this in the past, it made me forgot what object the `net` and `address` attributes live on.  Having `props` be the arg makes me instantly identify that it's react props coming through, not state (for example) or another manually passed object.

Reply:
In this case the `mapXtoProps` name gives it away :P Or is that a wrong assumption